src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] result = readFile(fromFileName);
+        try (FileWriter fileWriter = new FileWriter(toFileName)) {
+            for (String line : result) {
+                fileWriter.write(line + System.lineSeparator());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write the file" + toFileName, e);
+        }
+    }
 
+    public static String[] readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        int numberOfPurchases = 0;
+        int numberOfSupply = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String fileStr;
+            while ((fileStr = bufferedReader.readLine()) != null) {
+                stringBuilder.append(fileStr).append(",");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file " + fromFileName, e);
+        }
+        String[] cloneBuilder = stringBuilder.toString().split(",");
+        for (int i = 0; i < cloneBuilder.length; i++) {
+            if (cloneBuilder[i].equalsIgnoreCase("supply")) {
```

Comment:
Consider making `supply`, `buy` literals the constants

Reply:
when I put these values into the class fields. I get an error in the test 
org.junit.ComparisonFailure: Calling the getStatistic() method repeatedly returned incorrect results. 
if I make them constants, I can't change them afterwards.

Reply:
@OlexVoit Could you please provide some more info, where exactly are you going to change those literals?